docs/catalogue/cat_network-data-manager.md

```diff
@@ -1,58 +1,56 @@
 # CDM / Network data manager
 
-The Network data manager is responsible for uploading the common data model (CDM) to the Data Catalogue.
+The Network data manager is responsible for uploading the common data model (CDM) to the MOLGENIS catalogue.
 
-## Catalogue
+## MOLGENIS catalogue
 
 ### Define CDM metadata
 
-[MOLGENIS Data Catalogue](https://data-catalogue.molgeniscloud.org/catalogue/catalogue/#/variable-explorer/) (sometimes
-also called 'EMX2 catalogue') provides a framework to describe in detail: cohort metadata; definitions of the data
-variables collected (aka 'source variables'); and mappings to common data models (aka 'target variables). Its purpose is
-to facilitate pooled data analysis of multiple cohorts.
+[MOLGENIS catalogue](https://data-catalogue.molgeniscloud.org/catalogue/catalogue/#) provides a
+framework to describe in detail: metadata of different data resources, such as cohorts and data sources; definitions of 
```

Comment:
I would probably say "... data resources (such as cohorts and data sources); ..."